src/dg/dg.cpp

```diff
@@ -622,28 +642,138 @@ void DGBase<dim,real,MeshType>::assemble_cell_residual (
+                    } 
+
+                    const dealii::FEFaceValues<dim,dim> &fe_values_face_int = fe_values_collection_face_int.get_present_fe_values();
+                    const dealii::FEFaceValues<dim,dim> &fe_values_face_ext = fe_values_collection_face_ext.get_present_fe_values();
+
+                    const dealii::FESystem<dim,dim> &neigh_fe_ref = fe_collection[fe_index_neigh_cell];
+                   // const unsigned int neigh_cell_degree = neigh_fe_ref.tensor_degree();
+                    const unsigned int n_dofs_neigh_cell = neigh_fe_ref.n_dofs_per_cell();
+
+                    dealii::Vector<double> neighbor_cell_rhs (n_dofs_neigh_cell); // Defaults to 0.0 initialization
+
+                    const auto metric_neighbor_cell = high_order_grid->dof_handler_grid.begin_active();
+                    metric_neighbor_cell->get_dof_indices(neighbor_metric_dofs_indices);
+
+                    const real penalty = evaluate_penalty_scaling (current_cell, iface, fe_collection);
+
+              //      fe_values_collection_volume_neigh.reinit (neighbor_cell, i_quad, i_mapp, fe_index_neigh_cell);
+               //     const dealii::FEValues<dim,dim> &fe_values_volume_neigh = fe_values_collection_volume_neigh.get_present_fe_values();
+
+              //      if ( compute_dRdW ) {
+              //         assemble_face_term_implicit (
+              //                                     iface, neighbor_face_no, 
+              //                                     fe_values_face_int, fe_values_face_ext,
+              //                                     fe_values_volume, fe_values_volume_neigh,
+              //                                     penalty,
+              //                                     current_dofs_indices, neighbor_dofs_indices,
+              //                                     current_cell_rhs, neighbor_cell_rhs);
+              //     } else {
+    const dealii::types::global_dof_index neighbor_cell_index = neighbor_cell->active_cell_index();
+                if(!this->all_parameters->use_weak_form 
+                    && this->all_parameters->ode_solver_param.ode_solver_type == Parameters::ODESolverParam::ODESolverEnum::explicit_solver )//only for strong form explicit
+                {
+                    assemble_face_term_explicit (
+                        iface, neighbor_face_no, 
+                        current_cell,
+                        current_cell_index,
+                        neighbor_cell_index,
+                        poly_degree, grid_degree,
+                        fe_values_face_int, fe_values_face_ext,
+                        penalty,
+                        current_dofs_indices, neighbor_dofs_indices,
+                        current_metric_dofs_indices, neighbor_metric_dofs_indices,
+                        current_cell_rhs, neighbor_cell_rhs);
+
+                }
+                else {
+                //implicit weak 1D per boundary
```

Comment:
Remove empty `else` statement and simply add a comment explaining the else condition in the `if` statement above

Reply:
I decided to leave it because 1D weak form periodic BC should be added there in future.